src/common/middlewares/sync.js

```diff
@@ -2,12 +2,15 @@ import {REDUX_ACTION} from '../actions/runtime';
 
 
 
+const DefaultRegister = chrome.runtime.onMessage.addListener.bind(chrome.runtime.onMessage);
+const DefaultSend = chrome.runtime.sendMessage.bind(chrome.runtime);
+
 /**
  *	Gathers actions from other stores and passes it down to
  *	the next middlewares.
  *	This should be the first middleware in the chain.
  */
-export const createGatherMiddleware = (name, register = ::chrome.runtime.onMessage.addListener) =>
```

Comment:
Those binding operators are still not part of the language and might be confusing.